Package metadata example for Email notification steps. Add a link to the OctoStache unit test file with examples (#1199)
OctopusDeploy/docs: docs/projects/variables/variable-substitutions.md

| `Octopus.Template.Each.First` | `"True" if the element is the first in the collection`, otherwise "False" |
| `Octopus.Template.Each.Last`  | "True" if the element is the last in the collection, otherwise "False" |

> ### More Examples

Reviewer: ```suggestion
### Further examples
```

> If you're struggling with a specific syntax or OctoStache construction, you can find many more great examples in the unit tests defined for the library on GitHub:

Reviewer: ```suggestion
If you're struggling with a specific syntax or OctoStache construct, you can find more examples in the unit tests defined for the library on GitHub:
```

> [OctoStache Tests UsageFixture](https://github.com/OctopusDeploy/Octostache/blob/master/source/Octostache.Tests/UsageFixture.cs)

Reviewer: ```suggestion
[OctoStache Tests UsageFixture](https://github.com/OctopusDeploy/Octostache/blob/master/source/Octostache.Tests/UsageFixture.cs).
```
